src/app/dynamic/enterprise/quotas/project-quota-dialog/component.ts

```diff
@@ -52,6 +56,11 @@ export class ProjectQuotaDialogComponent implements OnInit, OnDestroy {
   projects: Project[] = [];
   selectedProject: Project;
   selectedQuota: QuotaDetails;
+  projectControl: FormControl<Record<ComboboxControls.Select, string>>;
+  // map of project id and count of its name duplicates
```

Comment:
I think there's no need for this comment and the one below because property names are self explanatory.